go/tools/gazelle/config/config.go

```diff
@@ -47,6 +47,10 @@ type Config struct {
 
 	// DepMode determines how imports outside of GoPrefix are resolved.
 	DepMode DependencyMode
+
+	// Log is used to print errors and warnings to the user. It should be used
```

Comment:
I am not sure what this gets you over just using
logs := &bytes.Buffer{}
log.SetOutput(logs)
in the tests?